src/controllers/Search-Controller.coffee

```diff
@@ -59,6 +59,9 @@ class SearchController
 
         logger?.info {user: @.req.session?.username, action:'show', queryId: queryId, filters:filters}
 
+        if (not queryId?)
```

Comment:
I realized that when the GraphDB is down or unavailable and you go to /show/, you do not get results and the queryId becomes undefined.
Is there a better option to check if it is unavailable from the Search controller?